unforbid.lua

```diff
@@ -2,10 +2,10 @@
 
 local argparse = require('argparse')
 
-local function unforbid_all(include_unreachable, quiet)
+local function unforbid_all(include_unreachable, include_tattered, quiet)
     if not quiet then print('Unforbidding all items...') end
 
-    local citizens = dfhack.units.getCitizens(true)
+    local citizens = dfhack.units.getCitizens()
```

Comment:
You can change this to `false` if you think you have a case for it, but don't remove the parameter. It causes a crash when in debug mode.